app/src/main/java/com/duckduckgo/app/onboarding/ui/page/DefaultBrowserPage.kt

```diff
@@ -63,6 +63,13 @@ class DefaultBrowserPage : OnboardingPageFragment() {
         super.onAttach(context)
     }
 
+    override fun setUserVisibleHint(isVisibleToUser: Boolean) {
```

Comment:
We are still not using the latest version of fragment-ktx but this method has been deprecated in 1.1.0 (https://developer.android.com/jetpack/androidx/releases/fragment). Probably not something for this PR but maybe create a LHF task?

Reply:
Good idea, internal task created here https://app.asana.com/0/414730916066338/1158589410650834